address_book.rb

```diff
@@ -1,13 +1,26 @@
+require 'csv'
+require './person.rb'
+
 class AddressBook
   def initialize(csv_path)
+    @people = Array.new
 
+    CSV.foreach(csv_path, headers: true) do |row|
+      @people = @people << Person.new(row[0], row[1], row[2], row[3])
```

Comment:
Buarada `@people =`  demene gerek yok aslinda. ```<<``` methodu @people'in kendisini guncelliyor